CONTRIBUTING.md

```diff
@@ -50,7 +50,7 @@ GreptimeDB uses the [Apache 2.0 license](https://github.com/GreptimeTeam/greptim
 
 - To ensure that community is free and confident in its ability to use your contributions, please sign the Contributor License Agreement (CLA) which will be incorporated in the pull request process.
 - Make sure all your codes are formatted and follow the [coding style](https://pingcap.github.io/style-guide/rust/).
-- Make sure all unit tests are passed.
+- Make sure all unit tests are passed (using `cargo test --workspace` or nextest `cargo nextest run`).
```

Comment:
What about adding a link to nextest? https://nexte.st/index.html

```suggestion
- Make sure all unit tests are passed (using `cargo test --workspace` or [nextest](https://nexte.st/index.html) `cargo nextest run --workspace`).
```

Reply:
Link is a good idea. However, per [nextest docs](https://nexte.st/#quick-start) `--workspace` flag is not needed. Could you pls edit your suggestion?)

Reply:
Fixed it.